src/compojure/api/logging.clj

```diff
@@ -0,0 +1,20 @@
+(ns compojure.api.logging
+  (:require [clojure.string :as str]))
+
+;; default to console logging
+(defn log! [level x & more]
```

Comment:
This is always defined and the redefined later if c.t.l is available? I think it would be clearer if this was only defined when c.t.l is not available.